node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go

```diff
@@ -568,25 +652,99 @@ func ConvertScsiIdToNguid(scsiId string) string {
 	return finalNguid
 }
 
-func (o GetDmsPathHelperGeneric) WaitForDmToExist(volumeUuid string, volumeNguid string, maxRetries int, intervalSeconds int) (string, error) {
-	formatTemplate := strings.Join([]string{"%d", "%w"}, mpathdSeparator)
+func (o GetDmsPathHelperGeneric) GetMpathdOutput(volumeIdVariations []string,
+	multipathdCommandFormatArgs []string) (string, error) {
+	return o.WaitForDmToExist(volumeIdVariations, WaitForMpathRetries,
+		WaitForMpathWaitIntervalSec, multipathdCommandFormatArgs)
+}
```

Comment:
I'm not sure we need `GetMpathdOutput`. if you think the name is better, you can rename `WaitForDmToExist`, though I'm not sure it's name is better

Reply:
I don't fully understand you, why do you think that we don't need `GetMpathdOutput`, and think of a better name for which function? `WaitForDmToExist`?